recipes/r-tsdyn/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set version = '11.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tsdyn
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/tsDyn_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/tsDyn/tsDyn_{{ version }}.tar.gz
+  sha256: 54d2e104cc8b40b9684feb9fb3e8c32c0167b8a6e99d5e0b4cb8d2a9f1db9c32
+
+build:
+  missing_dso_whitelist:
+    - "*/R.dll"  # [win]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-mass
+    - r-matrix
+    - r-foreach
+    - r-forecast
+    - r-mgcv
+    - r-mnormt
+    - r-nnet
+    - r-tseries
+    - r-tserieschaos
+    - r-urca
+    - r-vars
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-mass
+    - r-matrix
+    - r-foreach
+    - r-forecast
+    - r-mgcv
+    - r-mnormt
+    - r-nnet
+    - r-tseries
+    - r-tserieschaos
+    - r-urca
+    - r-vars
+
+test:
+  commands:
+    - $R -e "library('tsDyn')"           # [not win]
+    - "\"%R%\" -e \"library('tsDyn')\""  # [win]
+
+about:
+  home: https://github.com/MatthieuStigler/tsDyn/wiki
+  license: GPL-2.0-only
```

DESCRIPTION shows `GPL (≥ 2)`; also, let's provide direct link to repository:

```suggestion
  home: https://github.com/MatthieuStigler/tsDyn/wiki
  dev_url: https://github.com/MatthieuStigler/tsDyn
  license: GPL-2.0-or-later
```